src/core/Akka/IO/Inet.cs

```diff
@@ -221,6 +222,39 @@ public override void AfterConnect(Socket s)
                     //TODO: What is the .NET equivalent
                 }
             }
+
+            public class ByteBufferPoolSize : SocketOption
```

Comment:
The intent of this class is to explicitly set a Byte Buffer Pool size for an IO.TCP Connection without disrupting what is configured in HOCON as default.